### PR TITLE
Include crate features in docs

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Fixes https://github.com/gtk-rs/cairo/issues/285

### CODE REVIEW

**Comment by Hofer-Julian** [comment]
From my side this pull request would be finished.
If something needs to be changed please let me know.

File: src/lib.rs [hunk 1]
```diff
@@ -4,58 +4,61 @@
...
+//!   This library contains safe Rust bindings for [Cairo](https://www.cairographics.org/).
+//!   It is a part of [Gtk-rs](http://gtk-rs.org/).
+//!
 //! ## Crate features
 //!
 //! ### Default-on features
 //!
 //! * **use_glib** -
-//!   Explanation
+//!   Use [glib](https://gtk-rs.org/docs/glib/)
```
Comment (EPashkin) [inline src/lib.rs]: maybe "Use with glib" ?

**Comment by EPashkin** [comment]
@Hofer-Julian Thanks, 👍

File: src/lib.rs (hunk 1 above)
Comment (Hofer-Julian) [reply src/lib.rs]: Good point!

File: src/lib.rs [hunk 2]
```diff
@@ -2,6 +2,66 @@
 // See the COPYRIGHT file at the top-level directory of this distribution.
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
+//! # Cairo bindings
+//!
+//!   This library contains safe Rust bindings for [Cairo](https://www.cairographics.org/).
```
Comment (GuillaumeGomez) [inline src/lib.rs]: Why the two extra whitespaces at the beginning?

File: src/lib.rs [hunk 3]
```diff
@@ -2,6 +2,66 @@
...
+//!
+//! ### Default-on features
+//!
+//! * **use_glib** -
+//!   Use with [glib](https://gtk-rs.org/docs/glib/)
+//!
+//! ### Fileformat features
+//!
+//! * **png** -
+//!   Reading and writing PNG images
```
Comment (GuillaumeGomez) [inline src/lib.rs]: Why putting it on a newline?

File: src/lib.rs (hunk 2 above)
Comment (Hofer-Julian) [reply src/lib.rs]: Readability?
But if the convention differs, I will remove them.

File: src/lib.rs (hunk 3 above)
Comment (Hofer-Julian) [reply src/lib.rs]: No particular reason, I can remove them as well.

File: src/lib.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply src/lib.rs]: It's markdown so they'll get removed. And it makes the doc comment looking strange. It'd be nice to remove then if you don't mind. :)

File: src/lib.rs (hunk 3 above)
Comment (GuillaumeGomez) [reply src/lib.rs]: Yes please!

File: src/lib.rs (hunk 2 above)
Comment (Hofer-Julian) [reply src/lib.rs]: Done!
I removed quite a lot of whitespace.
If you want some back just tell me.

File: src/lib.rs (hunk 3 above)
Comment (GuillaumeGomez) [reply src/lib.rs]: What I meant was to write it this: `* **png** - Reading and writing PNG images`

Comment (Hofer-Julian) [reply src/lib.rs]: Done!
It looks indeed better

**Comment by Hofer-Julian** [comment]
I noticed that rust-fmt, made a small change.
Should I remove it?

**Comment by GuillaumeGomez** [comment]
No, don't worry about it. I'm looking at why the doc checker isn't happy.

**Comment by GuillaumeGomez** [comment]
Well whatever, merging. Thanks!

**Pull request merged**